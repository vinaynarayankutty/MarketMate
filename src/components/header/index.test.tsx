import React from 'react';
import {render} from '@testing-library/react-native';
import Header from './index';

describe('Header', () => {
  it('renders correctly', () => {
    const {toJSON} = render(<Header />);
    expect(toJSON()).toMatchSnapshot();
  });

  it('displays the correct heading text', () => {
    const {getByTestId} = render(<Header />);
    const headingElement = getByTestId('header-text');
    expect(headingElement.props.children).toBe('Upstox Holding');
  });
});
