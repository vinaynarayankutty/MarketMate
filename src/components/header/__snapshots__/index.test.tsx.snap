// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#7D017D",
      "borderWidth": 1,
      "height": 66.7,
      "justifyContent": "center",
      "paddingHorizontal": 12,
    }
  }
  testID="header-container"
>
  <Text
    style={
      {
        "color": "#FFFFFF",
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
    testID="header-text"
  >
    Upstox Holding
  </Text>
</View>
`;
